exchange/exchange_test.go

```diff
@@ -5750,3 +5750,182 @@ func TestSetSeatNonBid(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildMultiBidMap(t *testing.T) {
```

Comment:
Could you also a JSON test in `/exchangetest` similar to `multi-bids.json`, where we can test the case insensitivity there as well.

Reply:
Good catch! That extra test revealed another spot where case insensitivity was needed inside function `buildRequestExtMultiBid()`. Added `exchange/exchangetest/multi-bids-mixed-case.json`.